src-java/floodlightrouter-topology/floodlightrouter-storm-topology/src/main/java/org/openkilda/wfm/topology/floodlightrouter/model/RegionMappingStorage.java

```diff
@@ -13,12 +13,12 @@
  *   limitations under the License.
  */
 
-package org.openkilda.wfm.topology.statsrouter.service;
+package org.openkilda.wfm.topology.floodlightrouter.model;
 
-import org.openkilda.messaging.Message;
+public interface RegionMappingStorage {
```

Comment:
single implementation without any signs that any other will come in future, can't see any value from this

Reply:
In this specific case - this is part of visitor pattern implementation. I can say the same about all NB architecture and most part of switchmanager topology (and this is only what I can remember right now).